File: src/main/java/ar/edu/um/tesoreria/rest/model/Geografica.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.model;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
public class Geografica extends Auditable implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = 6459853826353848541L;

	@Id
	@Column(name = "geo_id")
	private Integer geograficaId;

	@Column(name = "geo_nombre")
	private String nombre = "";

	@Column(name = "geo_sinchequera")
	private Byte sinchequera = 0;

}

File: src/main/java/ar/edu/um/tesoreria/rest/kotlin/model/Geografica.kt
package ar.edu.um.tesoreria.rest.kotlin.model

import ar.edu.um.tesoreria.rest.model.Auditable
import jakarta.persistence.Entity
import jakarta.persistence.Table
import jakarta.persistence.Id
import jakarta.persistence.Column

@Entity
@Table
data class Geografica(

	@Id
	@Column(name = "geo_id")
	var geograficaId: Int? = null,

	@Column(name = "geo_nombre")
	var nombre: String = "",

	@Column(name = "geo_sinchequera")
	var sinChequera: Byte = 0

) : Auditable()